pkg/k8s/k8s.go

```diff
@@ -25,18 +25,19 @@ const (
 	defaultHTTPSPort     = int32(8443)
 	defaultHTTPSPortName = "https"
 
+	// Path and port to the healthcheck service
+	healthcheckPort        = 59999
+	healthcheckPath        = "/healthcheck"
+	healthcheckSidecarName = "nginx-healthchecker"
+
+	sidecarContainerImage = "tsuru/nginx-operator-sidecar:latest"
```

Comment:
We have to think about a way for configuring this image for deployments where the public registry is not available and no mirrors are configured. But this can be done later in another PR.

Reply:
Sure, I was thinking of it too. I will open an issue with that in order to keep it tracked.

Reply:
I opened #13